### ISSUE TITLE
Docker image for kube-linter

### ISSUE THREAD

**Issue by linuxbsdfreak** [stackrox/kube-linter#23]
Hi,

It would be nice if there would be a official docker image for kube-linter. Which would help in usage in CI/CD process .

Kevin

**Comment by viswajithiii** [comment]
Hi @linuxbsdfreak,

Thanks for filing this issue! This is definitely something we can do, but first I'd like to understand how you would like to use it to make sure we can handle that. Specifically, how do you want to pass the files to be linted to KubeLinter? Is your goal to be able to do something like `docker run -v <path-to-helm-files>:<path-in-container> <kube-linter-image>`?

**Comment by tdensmore** [comment]
That would be my use case... to avoid installing it locally, and allow it to be run from CI/CD tooling.

**Comment by linuxbsdfreak** [comment]
Hi @viswajithiii ,

Thanks. I am using tekton CI/CD to run tools to validate helm charts. I would like to use  kube-linter  as a tool packaged as a docker image  and run it against helm charts path or plain K8s manifests. That would be my use case. 

Kevin

**Comment by kreamkorokke** [comment]
Hi @linuxbsdfreak and @tdensmore 
That makes sense. We have added it to our roadmap and thanks again for bringing this up. I will loop this back to our main developer.

**Comment by jspiro** [comment]
Has anyone requested official support for running this using GitHub Actions? I can't find any such request. That would be our use-case, to run the above image in PRs using GitHub natively.

**Comment by viswajithiii** [comment]
> Has anyone requested official support for running this using GitHub Actions? I can't find any such request. That would be our use-case, to run the above image in PRs using GitHub natively.

Hi @jspiro, yes, we have discussed having an official GitHub Action before, but haven't tracked it in an issue as yet. I've created #96 to track this.

Even before that, though, it _should_ be possible to use KubeLinter without too much difficulty in a GitHub Action. We have an example [here](https://github.com/stackrox/contributions/tree/master/ci/GitHub/kube-linter) that may be helpful (it uses the binary directly instead of the image, which I think is preferable anyway since it avoids the dependency on Docker).
